openlogin/login-with-github/index.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from "@playwright/test";
+import { test } from "./index.lib";
+import { signInWithGithub } from "../../utils";
+
+test("Login with Github+Device", async ({ page, openloginURL, user }) => {
+  // Login with Github
+  await page.goto(openloginURL);
+  await page.click('button:has-text("Get Started")');
+  await page.click('[aria-label="Continue with existing GitHub"]');
+
+  try {
+    await page.waitForSelector("text=Reauthorization required", {
+      timeout: 10 * 1000,
+    });
+    await page.click('button:has-text("Authorize TorusLabs")');
```

Comment:
this text must've changed since then to `Authorize Web3Auth`

Reply:
it's still showing : Authorize TorusLabs